config/config.go

```diff
@@ -91,29 +91,48 @@ func (c *Config) parseLines(lines []string) {
 	}
 }
 
+// toLines read the whole Commands slice
+// and add at the end new strings values
+// then return the original lines plus new ones
 func (c *Config) toLines() []string {
 	lines := []string{}
 	for _, cm := range c.Commands {
-		lines = append(lines, "")
-		lines = append(lines, COMMAND_PREFIX+cm.Name)
-		lines = append(lines, PROPERTY_PREFIX+"description: "+cm.Description)
-		if cm.ID != "" {
-			lines = append(lines, PROPERTY_PREFIX+"id: "+cm.ID)
-		}
-		if cm.Usage != "" {
-			lines = append(lines, PROPERTY_PREFIX+"usage: "+cm.Usage)
-		}
-		if len(cm.Tags) > 0 {
-			lines = append(lines, PROPERTY_PREFIX+"tags: "+strings.Join(cm.Tags, ","))
-		}
-		if len(cm.Depends) > 0 {
-			lines = append(lines, PROPERTY_PREFIX+"depends: "+strings.Join(cm.Depends, ","))
-		}
-		if len(cm.Group) > 0 {
-			lines = append(lines, PROPERTY_PREFIX+"group: "+cm.Group)
+		for _, l := range cm.ToLines() {
+			lines = append(lines, l)
 		}
-		lines = append(lines, cm.Content)
 	}
+	return lines
+}
+
+// Byte takes the current lines from ToLines
```

Comment:
I would like to have *Config type* and its related methods on `config.go` while *Command* and its related methods on `command.go` file. Do you think you can split them?